connector-servicenow/src/main/java/com/redhat/cloud/notifications/connector/servicenow/ServiceNowCloudEventDataExtractor.java

```diff
@@ -6,23 +6,21 @@
 import jakarta.enterprise.context.ApplicationScoped;
 import jakarta.inject.Inject;
 import org.apache.camel.Exchange;
-import org.apache.commons.validator.routines.UrlValidator;
+import org.apache.hc.core5.http.URIScheme;
```

Comment:
Gentle reminder: we talked about replacing this lib with a simple constant. Is that still your plan?